components/gitpod-protocol/tsconfig.json

```diff
@@ -17,7 +17,7 @@
             "es2018",
             "dom",
             "ES2018.Regexp",
-            "DOM.AsyncIterable"
+            "DOM.Iterable"
```

Comment:
@filiptronicek `DOM.AsyncIterable` seems to be invalid, at least according to eslint and https://www.typescriptlang.org/tsconfig/#individual-library-components. Which one were you aiming for: `DOM.Iterable` or `ESNext.AsyncIterable`? :thinking:

Reply:
Looks like it compiles without this line, so let's remove it. I remember adding it because the in-ide linter said it's not a valid option and suggested me to use this `DOM.AsyncIterable`

Reply:
Done.